datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -355,6 +359,8 @@ export default function DefaultPreviewCard({
                         {hasGlossaryTerms && <TagTermGroup uneditableGlossaryTerms={glossaryTerms} maxShow={3} />}
                         {((hasGlossaryTerms && hasTags) || ((dataProduct || domain) && hasTags)) && <TagSeparator />}
                         {hasTags && <TagTermGroup uneditableTags={tags} maxShow={3} />}
+                        {hasAccess && <TagSeparator />}
+                        {hasAccess && <TagTermGroup uneditableAccess={access} maxShow={3} />}
```

Comment:
Remove this please. We do not want to accept changes to one of the most important components we have - the default preview card.

Reply:
done update